source/runbooks/daily-ops-tasks.html.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Daily ops tasks
+weight: 50.1
+last_reviewed_on: 2023-01-12
+review_in: 6 months
+---
+
+# Daily ops tasks
+
+This page explains how a day on the ops rotation works. In this role, our responsibility is to:
+
+| Responsibility | Why |
+| --- | --- |
+| Keep the service's integrity | Data cannot be corrupted. ("Service" is the entire "Refer and monitor an intervention", not just the API.) |
+| Secure the service | Security issues must be fixed immediately. |
+| Keep the service performant | Tolerances (CPU/memory/database storage/query performance) must be monitored. It cannot run out of pod limits, storage space or network bandwith. |
+| Reduce alert noise | Only actionable, real problems should create alerts. Everything else should be a dashboard. |
+
+
+### Checklist
+
+1. Check [Grafana][grafana] for any irregularities.
+
+1. Check [#interventions-alerts] for any AlertManager (Kubernetes) errors.
```

Comment:
Todo:

- Link to AlertManager and Prometheus guides on Cloud Platform.
- Link to where alerts are defined

Reply:
Another thing to define perhaps:

- Things e.g. `(sum(increase(nginx_ingress_controller_request_duration_seconds_bucket{exported_namespace="hmpps-interventions-prod",exported_service="hmpps-interventions-ui",le="+Inf"}[1m])) - sum(increase(nginx_ingress_controller_request_duration_seconds_bucket{exported_namespace="hmpps-interventions-prod",exported_service="hmpps-interventions-ui",le="10"}[1m]))) / (sum(increase(nginx_ingress_controller_requests{exported_namespace="hmpps-interventions-prod",exported_service="hmpps-interventions-ui"}[1m])))*100` what does this even mean

Reply:
not going to include the miro board -- that is linked on the confluence pages